index3.js

```diff
@@ -0,0 +1,43 @@
+// 0. Declare two variables "snackServings" and "guests" and assign them number values.
+let snackServings = 20;
+let guests = 10;
+let drinks = 10;
+let monkeys = 15;
+let bananas = 9;
+let monkeyfights = 20;
+
+// 1. Add snackServings and guests
+
+console.log(snackServings + guests);
+
+// 2. Subtract guests from snackServings. Then, subtract snackServings from guests.
```

Comment:
Try to also subtract snackServings from guests and print the result to the console!

Reply:
done it , thanks